packages/backend/src/graphql/schema.graphql

```diff
@@ -146,8 +169,36 @@ input UpdateAccountInput {
   routing: RoutingInput
 }
 
+type PeersConnection {
+  pageInfo: PageInfo!
+  edges: [PeerEdge!]!
+}
+
+type PeerEdge {
+  node: Peer!
+  cursor: String!
+}
+
+input CreatePeerInput {
+  disabled: Boolean
+  maxPacketAmount: UInt64
+  http: HttpInput!
+  asset: AssetInput!
+  stream: StreamInput
+  routing: RoutingInput!
+}
+
+input UpdatePeerInput {
+  id: String!
+  disabled: Boolean
+  maxPacketAmount: UInt64
+  http: HttpInput
+  stream: StreamInput
+  routing: RoutingInput
+}
```

Comment:
Another option is we could potentially have a single `PeerInput` object and then for the `updatePeer` mutation take in two args. 1. the `peerId` and 2. the `PeerInput`.

Don't have strong feels about it, just an observation

Reply:
The single `PeerInput` would need to either require fields when updating or allow fields to be optional when creating.